### DISCUSSION TITLE
Exposing a couple of things available to scripting api, such as projects

### DISCUSSION THREAD

**Discussion by solerante** [mapeditor/tiled#3515] [Ideas]
EDIT: please delete this as I have posted it in issues

Hi, great app! I'm trying to make a Tiled extension for working with maps from CDDA. Could it be possible to make some things available to the scripting API?

- Making the current **project path** available would be useful for building relative paths.
- Some way to **jump to a tile** in a tileset and/or tilemap. I made a search to find the tile I am looking for by it's custom property which in this case is the corresponding id in CDDA, but it doesn't seem like I can jump to and select that tileset and tile to see its properties. It became relevant because some tilesets are big with similar looking items.
EDIT: I found that this is already possible, using tileset.selectedTiles = [] (I thought it was read only for some reason)

edit: the project, if you are curious: https://github.com/solerante/tiled-cdda-map-extension

**Comment by bjorn** [comment]
> EDIT: please delete this as I have posted it in issues

Well, let's keep this topic, but for the record, it was reported as #3527 and added in Tiled 1.10. :-)
